syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -281,6 +281,9 @@ def mul_and_div(self, other, cmd):
         which is inherent to these operations in the fixed precision setting
         """
         changed_sign = False
+        if isinstance(other, float):
+            other = torch.tensor(other).fix_precision().child
```

Comment:
Q: Shouldn't we use the same precision as the ```self``` (if one has 4 and the other has 5, it takes automatically the lowest precision?)

Reply:
Your right!
`other` tensor must have same precision as `self`